webrender_traits/src/webgl.rs

```diff
@@ -973,6 +994,16 @@ impl WebGLCommand {
         chan.send(shader).unwrap();
     }
 
+    fn create_vertex_array(gl: &gl::Gl, chan: MsgSender<Option<WebGLVertexArrayId>>) {
+        let vao = gl.gen_vertex_arrays(1)[0];
+        let vao = if vao == 0 {
```

Comment:
Hmm, I unfortuntately don't remember why I added those checks.

That being said, we right now panic if we detect any gl error in the way, so I don't think this should be a safety problem.

This looks fine to me for now, since it's consistent with the rest of the code, if those checks need to be removed, and we need to check for `gl::get_error()` instead, we can do it in a separate patch.